lib/hydra/works.rb

```diff
@@ -64,21 +64,38 @@ module Works
     autoload :GenerateThumbnail,                   'hydra/works/services/generic_file/generate/thumbnail'
     autoload :PersistDerivative,                   'hydra/works/services/generic_file/persist_derivative'
 
-    # model validations
-    def self.collection? collection
+    # model validations 
+    def self.works_collection? collection
```

Comment:
Yes that works, but what about these types of tests:

old way:
        expect( Hydra::Works.works_collection? pcdm_coll ).to be false

new way?:
        expect( pcdm_coll.works_collection? ).to be false

both of these fail. Should I get rid of these tests?

On Mon, Aug 3, 2015 at 11:27 AM, Esmé Cowles <notifications@github.com>
wrote:

> In lib/hydra/works.rb
> <https://github.com/projecthydra-labs/hydra-works/pull/182#discussion_r36096531>
> :
>
> > @@ -64,21 +64,38 @@ module Works
> >      autoload :GenerateThumbnail,                   'hydra/works/services/generic_file/generate/thumbnail'
> >      autoload :PersistDerivative,                   'hydra/works/services/generic_file/persist_derivative'
> >
> > -    # model validations
> > -    def self.collection? collection
> > +    # model validations
> > +    def self.works_collection? collection
>
> The test should be able to check
>
> works_gwork.works_generic_work?
>
> instead of
>
> Hydra::Works.works_generic_work? works_gwork
>
> Does that work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra-labs/hydra-works/pull/182/files#r36096531>
> .
>